resync/api/monitoring_dashboard.py

```
@@ -386,40 +386,60 @@ async def disconnect(self, websocket: WebSocket):
+            await pubsub.unsubscribe(REDIS_CH_BROADCAST)
+
+        close_fn = getattr(pubsub, "close", None)
+        if close_fn is None:
+            return
+
+        with suppress(Exception):
+            maybe_awaitable = close_fn()
+            if asyncio.iscoroutine(maybe_awaitable):
+                await maybe_awaitable
+
     async def _pubsub_listener(self):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_pubsub_listener`

Add return type annotation: `None`

(ANN202)